requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django==1.8.2
+Django==1.8.3
```

Comment:
Is there anything in your pull request that requires 1.8.3?  Everyone should be settled now on 1.8.2 so I am hesitant to switch unless it is needed.  Other than that everything looks good to go.